packages/timeline-state-resolver/src/conductor.ts

```diff
@@ -398,21 +403,30 @@ export class Conductor extends EventEmitter<ConductorEvents> {
 			return Promise.reject(e)
 		}
 	}
+
 	/**
 	 * Creates an uninitialised device that can be referenced by the timeline and mappings.
 	 * @param deviceId Id used by the mappings to reference the device.
 	 * @param deviceOptions The options used to initalize the device
+	 * @param options Additional options
 	 * @returns A promise that resolves with the created device, or rejects with an error message.
 	 */
 	public async createDevice(
 		deviceId: string,
-		deviceOptions: DeviceOptionsAnyInternal
+		deviceOptions: DeviceOptionsAnyInternal,
```

Comment:
I am fairly certain that naming something, that is exported, with 'Internal' in its name is a bad code practise.
I know that it is unlikely to be you who have chosen the naming scheme to include 'Internal' in the name of the imported code - but if this could be improved upon now or later on, that would propably be good.

Reply:
Resolved Due to duplicate, from GitHub glitching out while writing it.